apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -80,13 +80,17 @@ export const commands = {
   },
 
   makeBurst(num, animation, effectName) {
+    var numSprites = this.getNumberOfSprites();
+    if (numSprites >= MAX_NUM_SPRITES) {
+      return;
+    }
     const behaviorFuncs = {
       burst: behaviorCommands.burstFunc,
       pop: behaviorCommands.popFunc,
       rain: behaviorCommands.rainFunc,
       spiral: behaviorCommands.spiralFunc
     };
-    num = Math.min(num, BIG_NUMBER_GUARD);
+    num = Math.min(num, MAX_NUM_SPRITES);
```

Comment:
Could we subtract `numSprites` here like in makeNumSprites?

```
num = Math.min(num, MAX_NUM_SPRITES - numSprites);
```

Reply:
Sure! The helper function definitely helps prevent this kind of typo!